modules/apps/portal/portal-tika-test/src/testIntegration/java/com/liferay/portal/tika/test/TikaRawMetadataProcessorTest.java

```diff
@@ -63,12 +68,18 @@ public void testGetRawMetadataMapWithXMatlab() throws Exception {
 		Assert.assertTrue(valueString.startsWith("application/javascript;"));
 	}
 
-	private InputStream _getInputStream(String fileName) {
-		Class<?> clazz = getClass();
+	private InputStream _getInputStream(String fileName) throws Exception {
+		File file = new File(fileName);
+
+		Files.write(
+			Paths.get(file.getPath()), _TEST_X_MATHLAB_FILE_CONTENT.getBytes());
 
-		return clazz.getResourceAsStream("dependencies/" + fileName);
+		return new FileInputStream(file);
```

Comment:
Should the file be deleted afterwards? (not sure)

Can the method return just this?
```java
	return new ByteArrayInputStream("function TikaRawMetadataProcessorTest() {}".getBytes());
```

Reply:
I agree with Marco; why create a file instead of returning a byte array input stream?

Reply:
I need the file to be called "test-x-matlab.js".

Reply:
I've upadated the PR, deletibng the file